Use time.Ticker in play-from-disk

Sean-Der opened the pull request:
time.Sleep is not precise, instead use a ticker.

Relates to golang/go#44343

Sean-Der commented:
@twometer mind trying this PR out? I have been trying a bunch of different things, time.Ticker is giving me the best results.

If it works for you/better then other timing patterns I am going to merge this with you as Co-author and then make an example-webrtc-applications PR

Twometer commented:
I've tried this PR, and the audio transmission works flawlessly, no lags whatsoever.
I timed the intervals and it seems the `time.Ticker` also uses some sort of "averaging out" system, these are the intervals I get:

![image](https://user-images.githubusercontent.com/26793103/129337963-606c453a-e570-42e4-8a88-cb18c759a62e.png)

Three or four ticks always average out to the expected 20ms/tick and it does so with much less jitter than the other timing pattern I've implemented in the test repository, so I think this might be the best solution!

jech commented:
Since this is an example, I think you should add a comment explaining the two reasons for the use of a ticker:

 * avoids accumulating skew (the Sleep didn't compensate for the time spent parsing the data);
 * works around latency issues with Sleep in older Go versions.

Other than that, LGTM.

Twometer commented:
I made a comment explaining the issue in [this example repo](https://github.com/Sean-Der/play-from-disk-h264). Maybe we can update the other example and comment to use the ticker, and then we'd have a consistent explanation and implementation across all examples.

Also, should we also update the `play-from-disk-renegotation` example?

Sean-Der commented:
Fixed all the suggestions, thank you so much!

* Updated `play-from-disk-renegotation`
* Added comment around Ticker usage to explain why we don't use Sleep anymore
* Added @Twometer as co-author on commits

As an aside @Twometer would you be interested in being added to the Pion organization? Would love your input/guidance on how we can make things better.

Twometer commented:
Sure, I'd love to help with the project if I can :)!
